packages/runner/injection/patches/cross-origin/cookies.ts

```diff
@@ -4,7 +4,7 @@ import {
   automationCookieToToughCookie,
   SerializableAutomationCookie,
 } from '@packages/server/lib/util/cookies'
-import type { Cookie as ToughCookie } from 'tough-cookie'
+import type { Cookie as ToughCookie } from '@packages/server/lib/util/cookies'
```

Comment:
This was the only use of this dep, to have a type defined. We’re already importing the server package this comes from though. Also the versions were mismatched.

Reply:
as long as it only a type this is fine, but we eventually want to break this pattern like we see above because nothing really should depend directly on the server. But that is a part of the ESM work